lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -158,7 +163,12 @@ def parseShaderAttributes(self):
                 pageLabel = 'Extra Shader Attributes'
             group = AEShaderLayout.Group(pageLabel)
             for name in nodeDef.GetPropertyNamesForPage(page):
-                if nodeDef.GetInput(name):
+                if Usd.GetVersion() < (0, 25, 5):
+                    nodeInput = nodeDef.GetInput()
+                else:
+                    nodeInput = nodeDef.GetShaderInput()
```

Comment:
The line above are in error: forgot to pass the `name` parameter. That is the cause of the unit test failure.

So much for doing code reviews eh? :)

Reply:
I fixed it for you.